### PR TITLE
JSON PDS4 label

### PR DESCRIPTION
**Pull request by tdddblog**
See Epic https://github.com/NASA-PDS/pds-api/issues/101
Corresponding pull request in "registry-api-service" project: https://github.com/NASA-PDS/registry-api-service/pull/63

### CODE REVIEW

File: swagger.yml [hunk 1]
```diff
@@ -99,13 +108,18 @@ paths:
...
+            # Codegen Workaround: codegen doesn't support multiple schemas. 
+            # Generic object has to be the first response type
+            application/pds4+xml:
+              schema:
+                type: object
             application/json:
               schema:
                 $ref: '#/components/schemas/product'
-            application/xml:
+            application/pds4+json:
```
Comment (jordanpadams) [inline swagger.yml:119]: @tdddblog is there are reason that we do not have the `application/pds4+json` available for all of the endpoints (e.g. `bundles/{lidvid}/collections`? it seems like this should apply everywhere.

Comment (jordanpadams) [reply swagger.yml:119]: @tdddblog ☝️

Comment (tdddblog) [reply swagger.yml:119]: @jordanpadams In the original story, only 3 endpoints were listed `/bundles/{lidvid}`, `/collections/{lidvid}`, and `/product/{lidvid}`. We also implemented `/bundles`, `/collections`, and `/products`. If we want this functionality for all endpoints we need to plan for it and estimate the effort based on total number of endpoints.

**Comment by jordanpadams** [comment]
merged from swaggerhub to main per dade5a0fe3d931f0bc8229003316dc3e30aed731, 043b23d957c6db4f42e52c3d769339c6257fb0d8, 42f54b99dfa1259f94d0f9467bd7f3ed6eee3e28

**Pull request closed without merging**